### ISSUE TITLE
openssl binaries are not given for testing

### ISSUE THREAD

**Issue by nav60** [deepbindiff/DeepBinDiff#14]
Hi,
Why Openssl exmaple is not included in the shared binaries? 
I have compiled both version for test example.  Please provide yours to verify the results

**Comment by qinzhengze** [comment]
Hello, I'm also trying to understand the code, are you still working on it? You can talk about it

**Comment by qinzhengze** [comment]
> Hi,
> Why Openssl exmaple is not included in the shared binaries?
> I have compiled both version for test example. Please provide yours to verify the results

Hello, I'm also trying to understand the code, are you still working on it? You can talk about it

**Comment by nav60** [comment]
Yes i worked on this project and taken diff of two binaries. what kind of query you have please ask here
